vulcand/registry.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	localEtcdProxy = "http://127.0.0.1:2379"
+	LocalEtcdProxy = "http://127.0.0.1:2379"
```

Comment:
See my example under implementation. This is required else it leads to `"http://127.0.0.1:2379"` duplication.